config/traject.rb

```diff
@@ -265,13 +265,13 @@
 to_field 'genre_tsim', extract_marc('650|*0|v:655|*0|abcvxyz:655|*7|abcvxyz')
 
 ## Genre facet (sidebar)
-to_field 'genre_facet', process_genre('650|*0|v:655|*0|a:655|*7|a')
+to_field 'genre_facet', process_genre('650|*0|v:655|*0|abcvxyz:655|*7|abcvxyz')
 
 ## Genre display
 to_field 'genre_display_ssm', process_genre('655|*0|abcvxyz:655|*7|abcvxyz')
 
 ## For genre links
-to_field 'genre_full_facet', extract_marc('650|*0|v:655|*0|abcvxyz:655|*7|abcvxyz'), trim_punctuation
+to_field 'genre_full_facet', process_genre('650|*0|v:655|*0|abcvxyz:655|*7|abcvxyz')
```

Comment:
I had to make this change to get the search to work when clicking the link.  Idk if there was a benefit to just using 'extract_marc' over the 'process_genre' method.

Reply:
it makes sense to make this change 👍🏻